plugins/shared-react/src/components/pipeline/HorizontalStackedBars.css

```diff
@@ -0,0 +1,23 @@
+.bs-shared-horizontal-stacked-bars {
+  --bar-gap: 3px;
+  display: block;
+  height: 100%;
+  overflow: hidden;
+  width: 100%;
+}
+.bs-shared-horizontal-stacked-bars.is-inline {
+  display: inline-block;
+}
+.bs-shared-horizontal-stacked-bars__bars {
+  display: flex;
+  flex-direction: row;
+  height: 100%;
+  outline: none;
+  width: calc(100% + var(--bar-gap));
+}
+.bs-shared-horizontal-stacked-bars__data-bar {
+  --bar-gap-neg: calc(var(--bar-gap) * -1);
+  box-shadow: inset var(--bar-gap-neg) 0 0 #fff;
```

Comment:
here `#fff` will always be white irrespective of theme i think 🤔

Reply:
without `#fff`
<img width="667" alt="Screenshot 2023-07-11 at 5 01 04 PM" src="https://github.com/janus-idp/backstage-plugins/assets/22490998/83084c35-0142-4fba-87b1-d2d843e650bb">

with `#fff`
<img width="667" alt="Screenshot 2023-07-11 at 5 01 16 PM" src="https://github.com/janus-idp/backstage-plugins/assets/22490998/d663d769-90bd-49b1-a4f8-e45fe6ad0cc7">